client/src/components/DataFiles/DataFilesProjectMembers/DataFilesProjectMembers.jsx

```diff
@@ -286,6 +286,7 @@ const DataFilesProjectMembers = ({
 };
 
 DataFilesProjectMembers.propTypes = {
+  projectId: PropTypes.string,
```

Comment:
Without projectId, was this broken previously. This seems more than refactor?

Reply:
Wasn't broken but included it for typechecking. I can remove it if its out of scope for this PR.